week3/index.js

```diff
@@ -1,15 +1,119 @@
-const Express = require('express')
-const bodyParser = require('body-parser')
+"use strict";
 
-const app = Express()
+const fs = require("fs");
+const express = require("express");
+const bodyParser = require("body-parser");
 
-app.use(bodyParser.json())
+const app = express();
+const todosFile = "todos.json";
+const PORT = 4000;
 
-const {list, create, update, remove} = require('./actions')
+app.listen(PORT, () => {
+    console.log(`Server listening on port ${PORT}`);
+});
+
+
+
+function handleRequest() {
```

Comment:
I'm curious why you put this in a function instead of just top-level code. It's not a bad thing - but normally the routes are just created in top-level code, since it only needs to happen once (so it doesn't need the repeatability of a function) and generally uses root-scope objects (like `app`).